docs/upload.rst

```diff
@@ -116,8 +116,8 @@ symbols.
    that URL in the HTTP POST.
 
 
-Upload by HTTP POST (payload < 2gb size)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Upload by HTTP POST
+~~~~~~~~~~~~~~~~~~~
```

Comment:
We no longer suggest picking one based on the size of the file. Instead we suggest breaking up large files into smaller ones regardless of the upload method.